README.md

```diff
@@ -381,6 +381,18 @@ Names of CSS [at-rules][at-rules] to allow that may have associated blocks
 containing style rules. At-rules like `media` and `keyframes` fall into this
 category. Names should be specified in lowercase.
 
+##### :css => :at_import_url_validator
```

Comment:
Could we call this `:import_url_validator` instead? I don't think the "at" adds anything useful to the name.